.golangci.yml

```diff
@@ -12,21 +12,21 @@ run:
 linters:
   disable-all: true
   enable:
-    # - deadcode
-    # - errcheck
+    - deadcode
+    - goconst
+    - goimports
     - gosimple
     - govet
     - ineffassign
-    - staticcheck
-    - unused
+    - misspell
+    # - staticcheck
```

Comment:
renable the ones we had before

Reply:
The ones we had result in a huge number of false positives that upstream geth doesn't care about. It's a huge ongoing maintenance burden and pulls us further away from upstream.

Reply:
would love to reevaluate this in a future PR. They do help finding bugs!

Reply:
https://github.com/celo-org/celo-blockchain/issues/1765